README.md

```diff
@@ -129,6 +130,51 @@ Properties can be provided by (order makes precedence):
   
 4. Mixed approach.
 
+### Validating properties
```

Comment:
defining section should be before validating ;)

Reply:
OK, I changed naming